moto/awslambda/models.py

```diff
@@ -1326,12 +1326,14 @@ def create_from_cloudformation_json(  # type: ignore[misc]
         account_id: str,
         region_name: str,
         **kwargs: Any,
-    ) -> "LambdaVersion":
+    ) -> "Optional[LambdaVersion]":
```

Comment:
same as before: what should moto do if someone tries to create a version of a lambda that doesn't exist? We can either return None or raise an error.

This same situation comes up a couple more times in this MR.